File: VB-Examples/10_Conversion/MarkDownToWordOrPDF.vb
Imports Spire.Doc

Namespace MarkDownToWordOrPDF
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Define the input file path relative to the current directory.
			Dim input As String = "..\..\..\..\..\..\Data\MarkDownFile.md"

			' Create a new Document object
			Dim doc As New Document()

			'Load .md file
			doc.LoadFromFile(input)

			'Save to .md file
			'doc.SaveToFile("output.md", Spire.Doc.FileFormat.Markdown);
			'Save to .docx file
			'doc.SaveToFile("output.docx", Spire.Doc.FileFormat.Docx);
			'Save to .doc file
			'doc.SaveToFile("output.doc", Spire.Doc.FileFormat.Doc);
			'Save to .pdf file
			doc.SaveToFile("output.pdf",FileFormat.PDF)

			' Dispose of the Document object
			doc.Close()

	
			Viewer("output.pdf")
		End Sub
		Private Sub Viewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS-Examples/10_Conversion/MarkDownToWordOrPDF.cs
using System;
using System.Windows.Forms;
using Spire.Doc;

namespace MarkDownToWordOrPDF
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Define the input file path relative to the current directory.
            string input = @"..\..\..\..\..\..\Data\MarkDownFile.md";

            // Create a new Document object
            Document doc = new Document();

            //Load .md file
            doc.LoadFromFile(input);

            //Save to .md file
            //doc.SaveToFile("output.md", Spire.Doc.FileFormat.Markdown);
            //Save to .docx file
            //doc.SaveToFile("output.docx", Spire.Doc.FileFormat.Docx);
            //Save to .doc file
            //doc.SaveToFile("output.doc", Spire.Doc.FileFormat.Doc);
            //Save to .pdf file
            doc.SaveToFile("output.pdf",FileFormat.PDF);

            // Dispose of the Document object
            doc.Close();

       
            Viewer("output.pdf");
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}
